Wangscape/OptionsManager.cpp

```diff
@@ -0,0 +1,54 @@
+#include "OptionsManager.h"
+
+#include <fstream>
+
+#include <boost/filesystem.hpp>
+
+#include <spotify/json.hpp>
+
+#include "codecs/OptionsCodec.h"
+
+OptionsManager::OptionsManager(std::string filename)
+{
+    std::ifstream ifs(filename);
+    if (!ifs.good())
+    {
+        throw std::runtime_error("Could not open options file");
+    }
+
+    std::string str{std::istreambuf_iterator<char>(ifs),
+                    std::istreambuf_iterator<char>()};
+
+    mOptions = spotify::json::decode<Options>(str.c_str());
+    mOptions.filename = filename;
+
+    // TODO(hryniuk): move it elsewhere
+    auto outputDirectory = mOptions.outputDirectory;
```

Comment:
Wangscape could always save the output in a current directory or ask for an another path. I don't understand why it cannot load and generate any images if it's able to do so. Wangscape uses SFML, so it could at least display generated images, if it cannot save it in a given directory.

Moreover, we could make it a feature (testing different parameters will be easier, I think) :wink:.